### PR TITLE
feat: use storybook for website

### PR DESCRIPTION
**Pull request by lwhiteley**
changes:
- simplify website by using storybook

run `npm start`

will research later how to deploy without commiting the docs folder the repo

### CODE REVIEW

File: src/stories/Dropdown.stories.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
+import React from 'react';
```
Comment (iambumblehead) [inline src/stories/Dropdown.stories.js:1]: Screenshot from 2020-10-26 10-30-33

This storybook addition is excellent. The default custom-arrow dropdown wraps down to the next line in local storybook.

Other than this single thing... this is a terrific MR that we should merge right away

**Review by iambumblehead — APPROVED** [review]
This is great please feel free to merge.

This looks great and will make development easier.

File: src/stories/Dropdown.stories.js (hunk 1 above)
Comment (lwhiteley) [reply src/stories/Dropdown.stories.js:1]: cool ill fix then merge

Comment (lwhiteley) [reply src/stories/Dropdown.stories.js:1]: its an existing problem as you said.. lets fix it in a dedicated Pull request

**Pull request merged**